src/Library/demos/Overlay Split View/code.rs

```diff
@@ -0,0 +1,27 @@
+use crate::workbench;
+use adw::prelude::*;
+use std::cell::RefCell;
+use std::rc::Rc;
+
+pub fn main() {
+    let overlay_split_view: Rc<RefCell<adw::OverlaySplitView>> = Rc::new(RefCell::new(
+        workbench::builder().object("split_view").unwrap(),
+    ));
```

Comment:
You should need neither Rc nor RefCell here.
GObjects already behave like `Rc<RefCell<T>>`